src/main/java/dota/buff/model/MatchDTO.java

```diff
@@ -0,0 +1,20 @@
+package dota.buff.model;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.experimental.FieldDefaults;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class MatchDTO {
+    long matchId;
+    String gameMode;
+    List<PlayerDTO> playerList;
+    List<HeroDTO> heroList;
+    boolean radiantWinner;
```

Comment:
It shouldn't be a boolean type here. Created separate story for changing this.

Reply:
+

Reply:
+

Reply:
Done